benchmarks/scripts/tests/deployed_contract_corpus_test.sh

```
@@ -0,0 +1,394 @@
+fn main() -> felt252 {
+    1
+}
+SRC
+}
+
+write_corpus_file() {
+  local path="$1"
+  local coverage="$2"
+  local dedupe_key="$3"
+  shift 3
+  local items_json="[$(IFS=,; echo "$*")]"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)